packages/expo-brightness/ios/BrightnessPermissionsRequester.swift

```diff
@@ -0,0 +1,15 @@
+import ExpoModulesCore
+
+class BrightnessPermissionsRequester: NSObject, EXPermissionsRequester {
```

Comment:
On iOS we don't have to ask for the permission to change the brightness, but afaik we do on Android, so this one is probably just for compatibility reasons. I'm not sure if we can remove that, but could you please add a `TODO` comment here?